src/modules/Builder/pages/BuilderApplet/BuilderApplet.tsx

```diff
@@ -120,7 +120,7 @@ export const BuilderApplet = () => {
   }, [defaultThemeId, isNewApplet]);
 
   useEffect(() => {
-    dispatch(themes.thunk.getThemes({}));
+    dispatch(themes.thunk.getThemes({ ordering: 'name' }));
```

Comment:
is Default theme will be chosen by default?
![image](https://github.com/ChildMindInstitute/mindlogger-admin/assets/116339462/8c800bf1-7866-43c6-a246-e33ed66714e9)